src/transformers/models/qwen2_moe/configuration_qwen2_moe.py

```diff
@@ -139,6 +143,7 @@ def __init__(
         norm_topk_prob=False,
         output_router_logits=False,
         router_aux_loss_coef=0.001,
+        mlp_only_layers=[],
```

Comment:
```suggestion
        mlp_only_layers=None,
```
it's bad practice! Let's default to [] if None when we do self.xxx = x !

Reply:
tuple works tho but more annoying to deal with

Reply:
it's right, my IDE also warned "Default Argument Value is mutable".
i searched for good practice to solve this is:
```
mlp_only_layers=None
......
self.mlp_only_layers = [] if mlp_only_layers is None else mlp_only_layers
```